test/e2e/walkthrough.go

```diff
@@ -113,6 +113,10 @@ var _ = framework.ServiceCatalogDescribe("walkthrough", func() {
 		err = util.WaitForClusterServiceClassToExist(f.ServiceCatalogClientSet.ServicecatalogV1beta1(), serviceclassID)
 		Expect(err).NotTo(HaveOccurred(), "failed to wait serviceclass to be ready")
 
+		By("Waiting for ClusterServicePlan to be ready")
+		err = util.WaitForClusterServicePlanToExist(f.ServiceCatalogClientSet.ServicecatalogV1beta1(), serviceplanID)
+		Expect(err).ShouldNot(HaveOccurred(), "serviceplan never became ready")
```

Comment:
The Gomega documentation recommends `NotTo` instead of `ShouldNot` when using `Expect`. That would keep it consistent with the rest of the test as well.

Reply:
Link to that recommendation? I was choosing the thing that 'read better' when I was looking at it. From what i could tell they were the exact same function.

Reply:
https://github.com/onsi/gomega/blob/1eecca0ba8e6f5ea5a431ce21d3aee2af0b4c90b/gomega_dsl.go#L288-L289